pkg/controllers/machinesync/machine_sync_controller.go

```diff
@@ -39,11 +47,28 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 )
 
+var (
+	// errAssertingCAPIAWSMachine is returned when we encounter an issue asserting a client.Object into a AWSMachine.
+	errAssertingCAPIAWSMachine = errors.New("error asserting the CAPI AWSMachine object")
+
+	// errAssertingCAPIPowerVSMachine is returned when we encounter an issue asserting a client.Object into a IBMPowerVSMachine.
+	errAssertingCAPIIBMPowerVSMachine = errors.New("error asserting the CAPI IBMPowerVSMachine object")
+)
+
 const (
+	reasonFailedToGetCAPIInfraResources    = "FailedToGetCAPIInfraResources"
+	reasonFailedToConvertMAPIMachineToCAPI = "FailedToConvertMAPIMachineToCAPI"
+	reasonFailedToUpdateCAPIInfraMachine   = "FailedToUpdateCAPIInfraMachine"
+	reasonFailedToCreateCAPIInfraMachine   = "FailedToCreateCAPIInfraMachine"
+	reasonFailedToUpdateCAPIMachine        = "FailedToUpdateCAPIMachine"
+	reasonFailedToCreateCAPIMachine        = "FailedToCreateCAPIMachine"
+
 	capiNamespace  string = "openshift-cluster-api"
 	mapiNamespace  string = "openshift-machine-api"
-	machineSetKind string = "MachineSet"
+	machineKind    string = "Machine"
 	controllerName string = "MachineSyncController"
+
+	messageSuccessfullySynchronized = "Successfully synchronized CAPI Machine to MAPI"
```

Comment:
Is this the correct way around?

Reply:
It is not and we had the same issue for the machineset_sync. Now fixed, Thanks.